assets/stylesheets/mixins.scss.tpl

```diff
@@ -14,3 +14,14 @@
 // Custom
 @import "variables";
 @import "helpers";
+
+// govuk-elements-sass
+@import "{{{govuk-elements-path}}}/helpers";
+@import "{{{govuk-elements-path}}}/reset";
+@import "{{{govuk-elements-path}}}/elements-typography";
+@import "{{{govuk-elements-path}}}/forms";
+@import "{{{govuk-elements-path}}}/buttons";
+@import "{{{govuk-elements-path}}}/details";
+@import "{{{govuk-elements-path}}}/lists";
+@import "{{{govuk-elements-path}}}/panels";
+@import "{{{govuk-elements-path}}}/tables";
```

Comment:
These aren't really mixins. We import the mixins as a standalone thing to use with custom styles.